lib/base.sh

```diff
@@ -12,8 +12,11 @@ ch_lib=${ch_bin}/../lib
 # Verbosity level; works the same as the Python code.
 verbose=0
 
+# Quiet level
+quiet=0
+
```

Comment:
Don't we have a single verbose level in the Python, rather than separate quiet and verbose? That seems tidier.